pkg/api/team_test.go

```diff
@@ -189,3 +190,72 @@ func TestTeamAPIEndpoint_CreateTeam_FGAC(t *testing.T) {
 		assert.Equal(t, http.StatusForbidden, response.Code)
 	})
 }
+
+// Given a team with a user, when the user is granted X permission,
+// Then the endpoint should return 200 if the user has accesscontrol.ActionTeamsWrite with teams:id:1 scope
+// else return 403
+func TestTeamAPIEndpoint_UpdateTeam_FGAC(t *testing.T) {
+	sc := setupHTTPServer(t, true, true)
+	sc.db = sqlstore.InitTestDB(t)
+	_, err := sc.db.CreateTeam("team1", "", 1)
+
+	require.NoError(t, err)
+
+	setInitCtxSignedInViewer(sc.initCtx)
+
+	input := strings.NewReader(fmt.Sprintf(createTeamCmd, 1))
```

Comment:
nit: maybe worth adding an `updateTeamCmd` even though it's the same or renaming `teamCmd`? Just for readability.